kafka-ui-react-app/src/components/Topics/Topic/Details/Messages/Filters/__tests__/InfoModal.spec.tsx

```diff
@@ -0,0 +1,46 @@
+import userEvent from '@testing-library/user-event';
+import { screen } from '@testing-library/react';
+import AddFilter, {
+  FilterModalProps,
+} from 'components/Topics/Topic/Details/Messages/Filters/AddFilter';
+import { render } from 'lib/testHelpers';
+import React from 'react';
+
+const editFilterMock = jest.fn();
+
+const setupComponent = (props: Partial<FilterModalProps> = {}) =>
+  render(
+    <AddFilter
```

Comment:
why are we rendering AddFilter in InfoModal spec  instead of infoModal ?

Reply:
Because I need to check the button works in AddFilter modal